### PR TITLE
Added support to assign attached mac address interface id and port options.

### PR DESCRIPTION
**Pull request by markleehamilton**
Updated code to pass pep8 v1.6.2 pylint v0.25.0.

### CODE REVIEW

File: network/openvswitch_port.py [hunk 1]
```diff
@@ -47,44 +54,188 @@
...
+        description:
+            - Used when port is created to set the external_ids:iface-id.
+    attached-mac:
+        required: false
+        description:
+            - MAC address when port is created using external_ids:attached-mac.
+    set:
+        required: false
+        description:
+            - Additional set options to apply to the port
```
Comment (bcoca) [inline network/openvswitch_port.py]: need version_added for new options

File: network/openvswitch_port.py [hunk 2]
```diff
@@ -47,44 +54,188 @@
         default: 5
         description:
             - How long to wait for ovs-vswitchd to respond
+    iface-id:
+        required: false
```
Comment (bcoca) [inline network/openvswitch_port.py]: when required: false, default must be specified

File: network/openvswitch_port.py [hunk 3]
```diff
@@ -17,6 +19,11 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this software.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Portions copyright @ 2015 VMware, Inc.  All rights reserved.
```
Comment (bcoca) [inline network/openvswitch_port.py]: keep copyright with existing above, also not sure 'All rights reserved' works with GPL code

Comment (markleehamilton) [inline network/openvswitch_port.py]: I asked this specific question to our lawyers. They say this is standard and does not conflict with GPL.
-Mark

File: network/openvswitch_port.py (hunk 2 above)
Comment (markleehamilton) [inline network/openvswitch_port.py]: Brian,
What is the process here? How should I address these changes?
-Mark

Comment (bcoca) [inline network/openvswitch_port.py]: just update the code and push again, the PR will have an additional commit (or you can rebase and force push)

**Comment by markleehamilton** [comment]
I believe I've address all of your comments, bcoca.

**Pull request merged**